woocommerce/woocommerce-gutenberg-products-block
# Fix Global Styles Block Icon Color for Mini Cart

This PR updates the Mini Cart icon under the Blocks list in the Global Styles customizer to be the same color as the rest of the icons in the list.

Fixes #7037 




### Screenshots



| Before | After |
| ------ | ----- |
|    ![CleanShot 2022-12-07 at 11 43 24@2x](https://user-images.githubusercontent.com/481776/206238987-045d0020-b07f-4861-8dc9-edcab9837849.png)    |    ![CleanShot 2022-12-07 at 11 35 35@2x](https://user-images.githubusercontent.com/481776/206239098-26363ce9-b984-468b-98ae-72739642d0b5.png)    |

### Testing



1. Make sure you have a blocks theme active (like Twenty Twenty-Two).
2. Navigate to Appearance > Editor (Beta) and go to the Global Styles customizer (click on the Styles button on the top toolbar and then the Blocks section towards the bottom - see screenshot below).
3. Scroll down to view the Mini Cart block and confirm the icon color is back like all of the others in the list.

![CleanShot 2022-12-07 at 11 58 12@2x](https://user-images.githubusercontent.com/481776/206242996-58e976b7-8a69-414d-8252-cff57099f3af.png)

### WooCommerce Visibility




### Changelog

> Fix Mini Cart icon color in Global Styles blocks customizer.
